src/NUglify.Tests/TestData/JS/Input/Literals/ObjectLiteral.js

```diff
@@ -41,7 +41,7 @@ var es6 = {
         // ES2015 computed property name
         [prop]: 'hey',
         ['b' + 'az']: 'there',
-        ['c' + x()]: 'hello',
+        [('c' + x())]: 'hello',
```

Comment:
Is the absence of these parenthesis invalid JS?
Asking from the PoV of you fixing a bug but you've changed the test input. if this is an additional case, can it be additional in the test.js instead of a replacement?